doc/releasenotes/16_0_0_summary.md

```diff
@@ -356,3 +356,18 @@ Creating a database with vttestserver was taking ~45 seconds. This can be proble
 In an effort to minimize the database creation time, we have changed the value of `tablet_refresh_interval` to 10s while instantiating vtcombo during vttestserver initialization. We have also made this configurable so that it can be reduced further if desired.
 For any production cluster the default value of this flag is still [1 minute](https://vitess.io/docs/15.0/reference/programs/vtgate/). Reducing this value might put more stress on Topo Server (since we now read from Topo server more often) but for testing purposes 
 this shouldn't be a concern.
+
+## Refactor
+
+### vttablet _vt sidecar schema maintenance refactor
+
+v16 changes the way we maintain the _vt schema. Instead of using `withddl` introduced in #6348 we use a declarative
```

Comment:
```suggestion
v16 changes the way we maintain vttablet's sidecar database schema. Instead of using `withddl` introduced in #6348 we use a declarative
```